.travis.yml

```diff
@@ -24,6 +25,5 @@ notifications:
     on_failure: change
 env:
 - secure: KG+szfdd5qhWb5C/kz05BND8wkEreqSOlZOWc9o2XXH2wFoabNUbA90mK2WeSR99gzoPqeydphrGwco+UPfoXcAwowhk3PzVOAZPhMlEX3G2znpeq8d17k4cQ/KoqpO1upKiADYqmxHkMaxjxb8JDOmeCqhhW5HYcOvTNUHdH2I=
-after_success:
-  git log -n 1 --format=%B | grep -iq "covr"; if [ $? -eq 0 ]; then echo "Checking test coverage with covr"; Rscript -e 'library(covr);coveralls()'; fi
-
+after_success: git log -n 1 --format=%B | grep -iq "covr"; if [ $? -eq 0 ]; then echo
```

Comment:
Did you mean to introduce a line break here? The ugly long line was originally intentional but maybe you've verified it still works WITH the line break? If not, please put this back the way it was and make a commit including `[covr]` both to check this is in working order and to update the test coverage info.